e2e/ansible/playbooks/hyperconverged/test-resize-jiva-volume/resize-volume.yml

```diff
@@ -0,0 +1,281 @@
+            state: unmounted
+          become: true
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+         # shell: umount -l "{{ mount_path.stdout }}"
+         # args:
+         #   executable: /bin/bash
+         # become: true
+         # register: unmount
+         # delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+
+
+        - name: Finding the iSCSI target name
+          shell: iscsiadm -m session | grep "{{ target_IP.stdout}}"| awk {'print $4'}
+          args:
+            executable: /bin/bash
+          become: true
+          register: iqn
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+          
+
+        - name: Logging out the iSCSI target 
+          open_iscsi:
+            login: no
+            target: "{{ iqn.stdout }}"
+          #shell: iscsiadm -m node -T "{{ iqn.stdout }}" -u
+          #args: 
+          #  executable: /bin/bash
+          become: true
+          delegate_to: "{{groups['kubernetes-kubeminions'].1}}"
+
+        - name: Getting the response of volume api
+          shell: curl http://"{{target_IP.stdout}}":9501/v1/volumes
+          args:
+            executable: /bin/bash
+          register: resp
+          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+          
+        - set_fact:
+            vol_id: "{{ (resp.stdout | from_json).data[0].id  }}"
+        
+        - name: Modify the capacity of volume
+          uri:
+            url: http://{{target_IP.stdout}}:9501/v1/volumes/{{vol_id}}?action=resize
+            method: POST
+            return_content: yes
+            body: {"name":"{{pv.stdout}}","size":"6 G"}
+            body_format: json
+            headers:
+              Content-Type: "application/json"
+          register: modify
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+         
+        - name: Get the list of pods
+          shell: kubectl get pods | grep {{ pv.stdout }} | grep rep | awk {'print $1'} | awk 'FNR == 1 {print}'
+          args:
+            executable: /bin/bash
+          register: rep1
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+
+        - name: Get the list of pods
+          shell: kubectl get pods | grep {{ pv.stdout }} | grep rep | awk {'print $1'} | awk 'FNR == 2 {print}'
+          args:
+            executable: /bin/bash
+          register: rep2
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+
+        - name: Get the list of pods
+          shell: kubectl get pods | grep {{ pv.stdout }} | grep rep | awk {'print $1'} | awk 'FNR == 3 {print}'
+          args:
+            executable: /bin/bash
+          register: rep3
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+
+        - name: Kill pods
+          shell: kubectl delete pod {{rep1.stdout}} {{rep2.stdout}} {{rep3.stdout}}
+          args:
+            executable: /bin/bash
+          delegate_to: "{{ groups['kubernetes-kubemasters'].0}}"
+ 
```

Comment:
Let us add a check to see if the replica pods are back up, running before proceeding